packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -107,6 +98,15 @@ export interface BaseServiceProps extends BaseServiceOptions {
    * Valid values are: LaunchType.ECS or LaunchType.FARGATE
    */
   readonly launchType: LaunchType;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service
+   *
+   * Valid values are: PropagatedTagSource.SERVICE, PropagatedTagSource.TASK_DEFINITION or PropagatedTagSource.NONE
+   *
+   * @default PropagatedTagSource.NONE
+   */
+  readonly propagateTags?: PropagatedTagSource;
```

Comment:
This is just moving the property, and I'm not seeing the rename?

Reply:
The renamed/new property already exists in Ec2ServiceProps and FargateServiceProps. My intent was to make it easier for customers as having 2 of the same property would be confusing especially when one of the props is not used in the case of Ec2Service and FargateService constructs.